_studio/shared/umc/io/umc_va/src/umc_va_linux.cpp

```diff
@@ -530,6 +530,13 @@ Status LinuxVideoAccelerator::Init(VideoAcceleratorParams* pInfo)
 
             va_res = vaGetConfigAttributes(m_dpy, va_profile, va_entrypoint, va_attributes, nattr);
             umcRes = va_to_umc_res(va_res);
+
+            //Get Gpu Priority
+            std::vector<VAConfigAttrib> va_Priorityattributes(1);
+            va_Priorityattributes[0].type = VAConfigAttribContextPriority;
+            va_res = vaGetConfigAttributes(m_dpy, va_profile, va_entrypoint, va_Priorityattributes.data(), 1);
+            umcRes = va_to_umc_res(va_res);
+            m_MaxContextPriority = va_Priorityattributes[0].value;
```

Comment:
Well, I would consider to move it here:
https://github.com/Intel-Media-SDK/MediaSDK/blob/48f48caf166e5231cf2e2f60a97f0ecd9279a48b/_studio/shared/src/libmfx_core_vaapi.cpp#L1367

Reply:
Done.